pool/pool.go

```diff
@@ -294,14 +285,14 @@ func newPool(ctx context.Context, options *BuilderOptions) (Pool, error) {
 			cliRes, err := createSessionTokenForDuration(ctx, c, options.SessionExpirationDuration)
 			if err != nil && options.Logger != nil {
 				options.Logger.Warn("failed to create neofs session token for client",
-					zap.String("address", address),
+					zap.String("address", addr),
 					zap.Error(err))
 			} else if err == nil {
 				healthy, atLeastOneHealthy = true, true
 				st := sessionTokenForOwner(ownerID, cliRes)
-				_ = cache.Put(formCacheKey(address, options.Key), st)
+				_ = cache.Put(formCacheKey(addr, options.Key), st)
```

Comment:
I think we should sign the `st` before put it to the cache (like it's done in line 652)

Reply:
https://github.com/nspcc-dev/neofs-sdk-go/pull/135/commits/d51e26eb147718e22126833a8ac4b59f4ed11cbb